test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,267 @@
+    // Now create the temp folder.  It should exists but create the folder if it is missing.
+    if (!fs.existsSync(tempFolderPath)) {
+      await utilities.createFolder(tempFolderPath);
+      await utilities.pause(1);
+    }
+  });
+
+  step('Run SFDX: Create Project', async () => {
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    // Selecting "SFDX: Create Project" causes the extension to be loaded, and this takes a while.
+
+    // Select the "Standard" project type.
+    let quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    expect(quickPicks.length).toEqual(3);
+    expect(await quickPicks[0].getLabel()).toEqual('Standard');
+    expect(await quickPicks[1].getLabel()).toEqual('Empty');
+    expect(await quickPicks[2].getLabel()).toEqual('Analytics');
+    await prompt.selectQuickPick('Standard');
+    await utilities.pause(1);
+
+    // Enter "TempProject-OrgCreationAndAuth" for the project name.
+    await prompt.setText(tempProjectName);
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    // Set the location of the project.
+    const input = await prompt.input$;
+    await input.setValue(tempFolderPath);
+    await utilities.pause(1);
+
+    // Click the OK button.
+    await utilities.clickFilePathOkButton();
+
+    // Verify the project was created and was loaded.
+    const sidebar = workbench.getSideBar();
+    const content = sidebar.getContent();
+    const treeViewSection = await content.getSection(tempProjectName.toUpperCase());
+    expect(treeViewSection).not.toEqual(undefined);
+
+    const forceAppTreeItem = await treeViewSection.findItem('force-app') as DefaultTreeItem;
+    expect(forceAppTreeItem).not.toEqual(undefined);
+
+    await forceAppTreeItem.expand();
+
+    // Yep, we need to wait a long time here.
+    await utilities.pause(10);
+  });
+
+  step('Run SFDX: Authorize a Dev Hub', async () => {
+    // This is essentially the "SFDX: Authorize a Dev Hub" command, but using the CLI and an auth file instead of the UI.
+    const workbench = await browser.getWorkbench();
+    await utilities.pause(1);
+
+    const authFilePath = path.join(projectFolderPath, 'authFile.json');
+    const terminalView = await utilities.executeCommand(workbench, `sfdx force:org:display -u ${EnvironmentSettings.getInstance().devHubAliasName} --verbose --json > ${authFilePath}`);
+
+    const authFilePathFileExists = fs.existsSync(authFilePath);
+    expect(authFilePathFileExists).toEqual(true);
+
+    await terminalView.executeCommand(`sfdx auth:sfdxurl:store -d -f ${authFilePath}`);
+
+    const terminalText = await utilities.getTerminalViewText(terminalView, 60);
+    expect(terminalText).toContain(`Successfully authorized ${EnvironmentSettings.getInstance().devHubUserName} with org ID`);
```

Comment:
Can you also add a line that we expect the org picker button to read "No Default Org Set"? In the spirit of having these tests capture the current reality of our commands, Authorizing a Dev Hub != Set that org as a default, and it would be helpful to remember it (and we see that the next test picks up on that assumption too)

Reply:
Done